miner/worker.go

```diff
@@ -393,10 +396,10 @@ func (w *worker) newWorkLoop(recommit time.Duration) {
 			timestamp = time.Now().Unix()
 			commit(false, commitInterruptNewHead)
 
-		/*case head := <-w.chainHeadCh:
-		clearPending(head.Block.NumberU64())
-		timestamp = time.Now().Unix()
-		commit(false, commitInterruptNewHead)*/
+		case head := <-w.chainHeadCh:
+			clearPending(head.Block.NumberU64())
+			timestamp = time.Now().Unix()
+		//commit(false, commitInterruptNewHead)
```

Comment:
why should we comment out `commit`?

Reply:
Timed block spawning has been changed to block spawning triggered by transaction.

Reply:
Add indent to the comment.